iquip/apps/dataviewer.py

```diff
@@ -393,13 +399,15 @@ class ButtonId(enum.IntEnum):
     def __init__(self, parent: Optional[QWidget] = None):
         """Extended."""
         super().__init__(parent=parent)
-        self.datasetEdit = QLineEdit(self)
-        self.datasetEdit.setPlaceholderText("Dataset")
+        self.datasetBox = QComboBox(self)
+        self.datasetBox.setPlaceholderText("Dataset")
```

Comment:
Does it work well?
In my local, this text looks ignored.

Reply:
Oh right, I don't see it neither!